src/services/RequestService.ts

```diff
@@ -59,4 +60,19 @@ export class RequestService {
       return await requestRepository.deleteRequest(request);
     });
   }
+
+  public async getMatchesByRequestId(params: UuidParam, getMatchesRequest: GetMatchesRequest): Promise<PostModel[]> {
+    return this.transactions.readOnly(async (transactionalEntityManager) => {
+      const requestRepository = Repositories.request(transactionalEntityManager);
+      let request;
+      if (getMatchesRequest.time == 0) {
```

Comment:
when would `time` be 0?

Reply:
i feel like i vaguely remember this being a design choice, but maybe add some documentation somewhere explaining when `time` can be 0 for future resell devs to not be confused

Reply:
If you are trying to get from this endpoint and you want to get all requests regardless of time, just make frontend make a request without passing a query parameter. That is clearer than setting the query parameter to 0 which has already caused confusion.